src/fireflyClient.ts

```diff
@@ -393,7 +395,7 @@ export class FireflyClient {
       this.getWallet(),
       this.maxBlockGasLimit,
       this.networkName,
-      this.getPublicAddress
+      publicAddress === undefined ? this.getPublicAddress : () => publicAddress
```

Comment:
This can be written as `publicAddress || this.getPublicAddress` no need of ternary condition

Reply:
I think they are not the same 
this returns a function which returns an address
```
publicAddress === undefined ? this.getPublicAddress : () => publicAddress
```
where as this returns an address value
```
publicAddress || this.getPublicAddress()
```